json-cache/service-worker.js

```diff
@@ -0,0 +1,43 @@
+// [Working example](/serviceworker-cookbook/json-cache/).
+
+var CACHE_NAME = 'dependencies-cache';
+
+self.addEventListener('install', function(event) {
+  // Perform install step:  loading each item into cache
+  event.waitUntil(
+    caches.open(CACHE_NAME)
+      .then(function(cache) {
+        console.log('[install] Caches opened');
+
+        return fetch('files-to-cache.json').then(function(response) {
+          return response.json();
+        }).then(function(files) {
+          console.log('[install] Adding files from JSON file: ', files);
+          return cache.addAll(files);
```

Comment:
It would be awesome if we could recognise the ServiceWorker API in the generated docco to provide the links to MDN when clicking on them, enhancement in issue #33. I will figure out.